draft-regext-rdap-extensions.md

```diff
@@ -265,18 +276,23 @@ be the extension identifier in cases where the extension defines only one object
         "rdap_level_0",
         "lunarNIC"  
       ],
-      "objectClassName": "lunarNIC author",
+      "objectClassName": "lunarNIC_author",
       "author":
       {
         "firstInitial": "R",
         "lastName": "Heinlein"
       }
     }
 
-Because the `objectClassName` is a string and [@!RFC9083] sets the precedent of using
-spaces in object class names (i.e. "ip network"), extensions may follow the same
-convention. It is RECOMMENDED that object class names be lower cased, ASCII characters
-that use the space character as a word separator.
+
+It is RECOMMENDED that object class names be lower cased, ASCII characters
+that use the "\_" (underscore) character as a word separator.
+Though `objectClassName` is a string and [@!RFC9083] does define
+one object class name with a space separator (i.e. "ip network") usage of the space character or
```

Comment:
"... (i.e. "ip network") usage of ..." -> "... (i.e. "ip network"), usage of ..." ?